framework/tests/value-null-strings-concatenation/Binaries.java

```diff
@@ -0,0 +1,350 @@
+import java.util.BitSet;
```

Comment:
Why did you add this test case?  It doesn't have string concatenations.

Reply:
It has one on line 79 ([link](https://github.com/typetools/checker-framework/blob/36cac07c83176088f7b589e200e222ec8cb13b49/framework/tests/value/Binaries.java#L79)), but that kind of assignment is overly tested in every other test, so I will remove `Binaries.java` from this test suite.